ncurses: add alacritty terminfo

tobiaspc opened the pull request:
Add terminfo file for the terminal emulator alacritty. 

https://github.com/alacritty/alacritty

Compile tested: x86-64 (APU2), ramips (ASUS RT-AX53U)
Run tested: ramips (ASUS RT-AX53U)
Size impact: 845 bytes

Related: https://github.com/openwrt/openwrt/issues/8902#issuecomment-1050997027

Signed-off-by: Tobias Hilbig <web.tobias@hilbig-ffb.de>

aparcar commented:
```
commit 8ace3fdecced34e70eb17b292427228077c9f5a0 (HEAD -> master)
Author: Tobi <web.tobias@hilbig-ffb.de>
Date:   Fri Feb 24 21:53:24 2023 +0100
```

SoB needs to be the same as your author name.

tobiaspc committed:
ncurses: add alacritty terminfo

Add terminfo file for the terminal emulator alacritty. 

https://github.com/alacritty/alacritty

Signed-off-by: Tobias Hilbig <web.tobias@hilbig-ffb.de>

tobiaspc commented:
Fixed SoB

aparcar commented:
To git.openwrt.org:openwrt/openwrt.git
   c9df2d5c64..888b207f1a  master -> master

Thanks
